### PR TITLE
Migrated Asset to H5

### PR DESCRIPTION
**Pull request by BenGalewsky**
Moved the Asset entity (owner_asset table) to H5.

Removed the base class to give this a recognizable id column. 

Added annotations and removed from medicade.hbn.xml

### CODE REVIEW

**Comment by jasonaowen** [comment]
Why am I listed as the author on this commit?

I have no memory of this place

File: psm-app/db/seed.sql [hunk 1]
```diff
@@ -1080,3 +1081,15 @@ CREATE TABLE licenses(
     REFERENCES specialty_types(code),
   attachment_id BIGINT
 );
+
+CREATE TABLE owner_assets(
+  owner_asset_id BIGINT PRIMARY KEY ,
+  type CHARACTER VARYING(255),
+  name CHARACTER VARYING(255),
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: These should be `TEXT` rather than `CHARACTER VARYING(255)`.

File: psm-app/db/seed.sql [hunk 2]
```diff
@@ -1080,3 +1081,15 @@ CREATE TABLE licenses(
...
+  address_id BIGINT
+    REFERENCES addresses(address_id),
+  ownership_typ_cd CHARACTER VARYING(2)
+    REFERENCES ownership_types(code),
+  ownership_info_id BIGINT,
+  idx INTEGER
+   ) ;
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: Can you please clean up the indentation and whitespace here?

File: psm-app/services/src/main/java/gov/medicaid/entities/Asset.java [hunk 3]
```diff
@@ -15,108 +15,86 @@
...
+public class Asset implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "owner_asset_id")
+    private long id;
 
     /**
      * The asset type.
      */
+    @Column(name ="ownership_typ_cd")
```
Comment (jasonaowen) [inline psm-app/services/src/main/java/gov/medicaid/entities/Asset.java]: `AssetType` is an `enum`, not an `@Entity` - and in particular, not an `OwnershipType`.

File: psm-app/db/seed.sql [hunk 4]
```diff
@@ -1080,3 +1081,15 @@ CREATE TABLE licenses(
...
   attachment_id BIGINT
 );
+
+CREATE TABLE owner_assets(
+  owner_asset_id BIGINT PRIMARY KEY ,
+  type CHARACTER VARYING(255),
+  name CHARACTER VARYING(255),
+  address_id BIGINT
+    REFERENCES addresses(address_id),
+  ownership_typ_cd CHARACTER VARYING(2)
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: Please rename this to `ownership_type_code`.

File: psm-app/db/seed.sql [hunk 5]
```diff
@@ -1080,3 +1081,15 @@ CREATE TABLE licenses(
...
+    REFERENCES ownership_types(code),
+  ownership_info_id BIGINT,
+  idx INTEGER
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: What is `idx`? I don't see any references to this column in the class.

File: psm-app/db/seed.sql [hunk 6]
```diff
@@ -1080,3 +1081,15 @@ CREATE TABLE licenses(
...
+
+CREATE TABLE owner_assets(
+  owner_asset_id BIGINT PRIMARY KEY ,
+  type CHARACTER VARYING(255),
+  name CHARACTER VARYING(255),
+  address_id BIGINT
+    REFERENCES addresses(address_id),
+  ownership_typ_cd CHARACTER VARYING(2)
+    REFERENCES ownership_types(code),
+  ownership_info_id BIGINT,
```
Comment (jasonaowen) [inline psm-app/db/seed.sql]: What is `ownership_info_id`? I don't see any references to this column in the class.

**Comment by BenGalewsky** [comment]
Closed as duplicate of #138

**Pull request closed without merging**